arch/arm/src/armv8-m/Kconfig

```diff
@@ -153,3 +153,19 @@ config ARMV8M_SYSTICK
 	depends on TIMER
 	---help---
 		Enable SysTick timer driver.
+
+config ARMV8M_TRUSTZONE_HYBRID
+	bool "Hybrid with secure and non-secure."
+	default n
+	depends on SMP
+	---help---
+		Enable the hybrid with secure and non-secure domain.
+		In SMP mode, you can enable this configuration when one
+		core supports secure and the other does not.
+
+config ARMV8M_TRUSTZONE_CPU_BITMASK
+	int "Security support bitmap for multicore."
```

Comment:
Maybe at least `hex` instead of `int`. I barely can imagine people setting mask as int